loan-calculator/src/index.js

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import 'bootstrap/dist/css/bootstrap.css';
```

Comment:
If we're dropping bootstrap, do we still want the `npm` packages in `package.json`?

Reply:
good catch. ill remove that now.